docs/markdown/Creating-releases.md

```diff
@@ -60,6 +60,14 @@ for example when done in CI that already does its own testing.
 So with `--no-tests` you can tell Meson "Do not build and test generated
 packages.".
 
+## Use `--allow-dirty` to override error when git repository contains uncommitted changes
+
+*Since 0.62.0* Instead of emitting a warning when a repository contains
+uncommitted changes, Meson will produce an error message notifying the
+user and immediately exit. If `--allow-dirty` is given as an option to
```

Comment:
Should no warning be given in this case if you are explicitly asking Meson to allow this?

Reply:
@tristan957 That's a good point and I can't argue with your logic. xD

I don't have strong feelings either way, I pretty much do whatever Eli tells me to do. :)